database/entity/0003-login_server_tables/LoginEmailOptIn.ts

```diff
@@ -6,21 +6,21 @@ export class LoginEmailOptIn extends BaseEntity {
   @PrimaryGeneratedColumn('increment', { unsigned: true })
   id: number
 
-  @Column({ name: 'user_id' })
+  @Column({ name: 'user_id', type: 'bigint', unsigned: true, nullable: false })
   userId: number
 
   @Column({ name: 'verification_code', type: 'bigint', unsigned: true, unique: true })
   verificationCode: BigInt
 
-  @Column({ name: 'email_opt_in_type_id' })
+  @Column({ name: 'email_opt_in_type_id', type: 'int', unsigned: true, nullable: false })
   emailOptInTypeId: number
 
-  @Column({ name: 'created', default: () => 'CURRENT_TIMESTAMP' })
+  @Column({ name: 'created', type: 'datetime', default: () => 'CURRENT_TIMESTAMP' })
```

Comment:
for a correct initialisation and setting of the ms, the type and initial value should be define with the accuracy of 3 like datetime(3) like it is described https://mariadb.com/kb/en/microseconds-in-mariadb
This is necessary and defined in gradido as standard for all created_at columns for correct sorting base on datetime columns especially to distingue entries created in the same second